codex-ui/src/vue/card/Card.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div :class="[$style.card, orientation === 'horizontal' ? $style['card--horizontal'] : '']">
+    <div :class="$style['card__cover']"></div>
+    <div :class="$style['card__footer']">
+      <Heading :level="3">{{ title }}</Heading>
+      <div :class="[$style['card__updatedAt'], 'text-ui-subtle']">{{ updatedAt }}</div>
+    </div>
+    <slot name="button"></slot>
```

Comment:
slot name is redundant here, since it is the only slot in a component. Also, there can be not only button but anything. So its better to remove name